packages/protocol/contracts/governance/interfaces/IValidators.sol

```diff
@@ -2,16 +2,10 @@
 pragma solidity >=0.5.13 <0.9.0;
 
 interface IValidators {
-  function registerValidator(
```

Comment:
I'd leave so
`function registerValidatorNoBls(bytes calldata ecdsaPublicKey) external returns (bool);` and 
```unction registerValidator(    bytes calldata,
    bytes calldata,
    bytes calldata
  ) external returns (bool);
  ```

Have the same behaviour (like in GovernanceSlasher), even if not it's inconsistent that `slashL2` got removed in favour of `slash`, but there `registerValidatorNoBls` is the one that's still here

Reply:
is this what you meant? commit 043f9707417617ab6f49ac6f12c64a6134fdaad1